test/test_quantization.py

```diff
@@ -607,11 +606,12 @@ def checkQuantized(model):
         model = quantize(model, test_only_eval_fn, self.img_data)
         checkQuantized(model)
 
+
 class ObserverTest(QuantizationTestCase):
     @given(qdtype=st.sampled_from((torch.qint8, torch.quint8)),
            qscheme=st.sampled_from((torch.per_tensor_affine, torch.per_tensor_symmetric)))
-    def test_observer(self, qdtype, qscheme):
-        myobs = Observer(dtype=qdtype, qscheme=qscheme)
+    def test_minmax_observer(self, qdtype, qscheme):
+        myobs = MinMaxObserver(dtype=qdtype, qscheme=qscheme)
```

Comment:
Have you rebased to tip? Please verify that you have changes from https://github.com/pytorch/pytorch/pull/24996/files and ensure that observer is scriptable.